educator/src/Dscp/Educator/Web/Student/Types.hs

```diff
@@ -114,6 +116,50 @@ signedSubmissionToRequest sigSub =
         , nsWitness = _ssWitness sigSub
         }
 
+---------------------------------------------------------------------------
+-- Buildable instances
+---------------------------------------------------------------------------
+
+instance Buildable (IsEnrolled) where
+    build (IsEnrolled{..}) =
+      "{ is enrolled = " +| unIsEnrolled |+
+      " }"
+
+instance Buildable (NewSubmission) where
+    build (NewSubmission{..}) =
+      "{ hash = " +| nsAssignmentHash |+
+      " }"
```

Comment:
IMO _at least_ for all `New*` we want all fields to be printed to understand what is going on just looking at logs.

Reply:
Is it ok now?

Reply:
Yep, looks great :+1: